packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1913,3 +1913,18 @@ function VM-Unzip-Recursively {
         }
     }
 }
+# Obtains the chocolatey install script path from a package, for example: C:\ProgramData\chocolatey\lib\capa.vm\tools\chocolateyinstall.ps1
+# Reads the tag with name tags from the nuspec file,
+# Returns the category if the tags exist in the nuspec
+function VM-Get-Category {
+    $installPath = $MyInvocation.MyCommand.Definition
+    $toolDir = "$(Split-Path -parent (Split-Path -parent $installPath))"
```

Comment:
According to [our documentation](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#tooldir) `$toolDir` should only be used for directory where the tool is actually installed. We need a different name here. I think you can also remove the `"` 
```suggestion
    $chocolateyToolDir = $(Split-Path -parent (Split-Path -parent $installPath))
```